src/app/App.tsx

```diff
@@ -0,0 +1,84 @@
+import { Component } from 'react';
+import { Header } from '@widgets/header';
+import { SearchPage } from '@pages/SearchPage';
+import { fetchItems, Item } from '@shared/api/items';
+import { ErrorBoundary } from '@shared/ui/ErrorBoundary/ErrorBoundary';
+
+import '@shared/styles/global.css';
+
+type State = {
+  items: Item[];
+  isFetching: boolean;
+  isResultEmpty: boolean;
+  userValue: string;
+  inputValue: string;
+  limit: string | number;
+  page: string | number;
+};
+
+export default class App extends Component {
+  state: State = {
+    items: [],
+    isFetching: false,
+    isResultEmpty: false,
+    userValue: '',
+    inputValue: '',
+    limit: 10,
+    page: 1,
+  };
+
+  handleInputName = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Try not to shorten names in your code. That is not a really good practice
```suggestion
  handleInputName = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Reply:
short naming has been fixed to full